app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -407,12 +410,40 @@ class LuciBuildService {
 
     final Build build = builds.first;
     final Map<String, Object>? properties = build.input!.properties;
+    final Map<String?, List<String?>> tags = Map<String?, List<String?>>.from(build.tags!);
     log.info('input ${build.input!} properties $properties');
-
-    final ScheduleBuildRequest scheduleBuildRequest =
-        await _createPostsubmitScheduleBuild(commit: commit, target: target, task: task, properties: properties);
-    final Build scheduleBuild = await buildBucketClient.scheduleBuild(scheduleBuildRequest);
-    return scheduleBuild;
+    log.info('input ${build.input!} tags $tags');
+    tags['trigger_type'] = <String>['check_run_manual_retry'];
+    try {
+      await updateTaskStatusInDatabase(
+        task = task,
+        taskDocument = taskDocument,
+        firestoreService = firestoreService,
+        datastore = datastore,
+      );
+    } catch (error) {
+      log.severe(
+        'updating task ${taskDocument.taskName} of commit ${taskDocument.commitSha} failure: $error. Skipping rescheduling.',
+      );
+      return;
+    }
+    tags['current_attempt'] = <String>[(taskDocument.attempts!).toString()];
+    log.info('Updated input ${build.input!} tags $tags');
+    final BatchRequest request = BatchRequest(
```

Comment:
I guess I am not understanding, is this really a postsubmit task that is run in presubmit for packages?

Reply:
Postsubmit task is for postsubmit CI. It shares the same github UI for checkrun view. You can view the postsubmit ones:
![Screenshot 2024-04-22 at 3 55 07 PM](https://github.com/flutter/cocoon/assets/54558023/4d4c1a0b-d35a-49d3-bcbe-723d20e78345)